tests/test_app/test_io_new.py

```diff
@@ -54,7 +80,7 @@ def test_write_seqs(fasta_dir, tmp_dir):
     out_data_store = DataStoreDirectory(
         tmp_dir / "test_write_seqs", mode=Mode.w, suffix="fasta"
     )
-    writer = io_app_new.write_seqs(out_data_store, format="fasta")
+    writer = io_app.write_seqs(out_data_store, format="fasta")
```

Comment:
Here , you have called old write_seqs.

Reply:
Actually, I changed the imports at the top of the file to
```python
from cogent3.app import io_new as io_app
```